requirements-dev.in

```diff
@@ -2,3 +2,4 @@ mypy
 pytest
 pre-commit
 black
+pyfakefs
```

Comment:
I would not recommend this package -- if you're reaching for this it indicates you're not properly fixturing.  use `tmp_path` pytest fixture for your paths and you should be good